src/Query/Builder.php

```diff
@@ -250,6 +253,30 @@ public function cursor($columns = [])
         throw new RuntimeException('Query not compatible with cursor');
     }
 
+    /**
+     * Die and dump the current MongoDB query
+     *
+     * @return never-return
+     */
+    #[Override]
```

Comment:
Keeping track to add this attribute where necessary: https://jira.mongodb.org/browse/PHPORM-146